examples/example_emscripten_wgpu/main.cpp

```diff
@@ -247,3 +255,11 @@ static void print_wgpu_error(WGPUErrorType error_type, const char* message, void
     }
     printf("%s error: %s\n", error_type_lbl, message);
 }
+
+static EM_BOOL on_canvas_size_changed(int /* event_type */, const EmscriptenUiEvent* /* ui_event */, void* user_data) {
+    double canvas_width, canvas_height;
+    emscripten_get_element_css_size(canvas_selector, &canvas_width, &canvas_height);
+    glfwSetWindowSize((GLFWwindow *)user_data, (int)canvas_width, (int)canvas_height);
+    emscripten_set_canvas_element_size(canvas_selector, canvas_width, canvas_height);
```

Comment:
Is there a particular reason to calling `emscripten_set_canvas_element_size` since `canvas_width` and `canvas_height` were extracted from `emscripten_get_element_css_size`?